### ISSUE TITLE
CMD+K / CTRL+K Shortcut and inputs not always captured by command bar plugin

### ISSUE THREAD

**Issue by Sebobo** [Sebobo/Shel.Neos.CommandBar#8]
Issues: 

- [x] Shortcut to open command bar not working properly inside guest frame.
- [ ] CKEditor maps CMD+K to the link editor
- [x] Arrow navigation should work in input and when the dialog is focused
- [x] On Windows and Linux CTRL+K opens the search bar in the browser (this can be prevented somehow as other websites do that)

**Comment by PRGfx** [comment]
Ctrl+K conflicts with focusing the address bar in [chrome](https://support.google.com/chrome/answer/157179?hl=en) and [firefox](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_search) on Windows and Linux, so while the command bar opens, the focus is gone. If that can't be prevented, maybe the shortcut could be configurable?

**Comment by Sebobo** [comment]
Oh thx for reporting the issue! 
I had several testers, but apparently all were Mac users. I was quite sure that CTRL+L would be the focus of the address bar like CMD+L is on the Mac. 
I wonder why all those command bar plugins then call themselves CMD+K 😔
Making it configurable wouldn't help much as the Neos backend will be used by a variety of users. I somehow must detect the available shortcut...

**Comment by Sebobo** [comment]
@PRGfx so which is the proper letter for this kind of action on Windows/Linux?

**Comment by Sebobo** [comment]
Yes I was just told that it can be prevented and several websites apparently do that. So I will do that now too ;)

**Comment by Sebobo** [comment]
Hopefully resolved now, sadly cannot test it properly right now with my Windows VM

**Comment by PRGfx** [comment]
Unfortunately it doesn't seem fixed with 0.6.1 as far as I can tell.
I tried looking into this, but the parts changed in 9935282 are working on `event.metaKey`, which shouldn't trigger for `Ctrl+K`, so I can't really see where the Commandbar comes from..
(Trying to build it locally, the compiler was complaining about missing type declarations btw.)

**Comment by Sebobo** [comment]
OK, now with 0.7.2 at least this problem should be done. 

The behaviour in the guest frame also improved a bit. The link editor still opens when inline editing, but another shortcut press will open the command bar, so there is a way to reach it. On MacOS Ctrl+K works fine though. 
So not sure now whether and how to prevent the problem in CK. If users use the link hotkey then it would not be nice to prevent them from using it. So we could leave it like this for now and add a note to the readme ...
